plugins/crashes/frontend/public/javascripts/countly.views.js

```diff
@@ -1,3756 +1,435 @@
+            },
+            mobileDiagnostics: function() {
+                return this.$store.getters["countlyCrashes/crashgroup/mobileDiagnostics"];
+            },
+            mobileMetrics: function() {
+                return this.$store.getters["countlyCrashes/crashgroup/mobileMetrics"];
+            },
+            chartData: function() {
+                return this.$store.getters["countlyCrashes/crashgroup/chartData"](this.$data.chartBy);
+            },
+            chartByOptions: function() {
+                return this.$store.getters["countlyCrashes/crashgroup/chartByOptions"];
+            },
+            crashes: function() {
+                return this.$store.getters["countlyCrashes/crashgroup/crashes"];
+            }
+        },
+        methods: {
+            refresh: function() {
+                return this.$store.dispatch("countlyCrashes/crashgroup/refresh");
+            },
+            handleCommentCommand: function(command, comment) {
+                if (command === "edit-comment") {
+                    this.$data.commentInput = comment.text;
+                    this.$data.commentBeingEdited = comment;
                 }
-                else if (val === "crash-hide") {
-                    CountlyHelpers.confirm(jQuery.i18n.prop("crashes.confirm-action-hide", self.selectedCrashesIds.length), "red", function(result) {
-                        if (!result) {
-                            return true;
-                        }
-                        countlyCrashes.hide(self.selectedCrashesIds, function(data) {
-                            if (!data) {
-                                CountlyHelpers.alert(jQuery.i18n.map["crashes.try-later"], "red");
-                            }
-                            else {
-                                self.resetSelection(true);
-                            }
-                        });
-                    });
+                else if (command === "delete-comment") {
+                    this.$store.dispatch("countlyCrashes/crashgroup/deleteComment", comment._id);
                 }
-                else if (val === "crash-resolving") {
-                    CountlyHelpers.confirm(jQuery.i18n.prop("crashes.confirm-action-resolving", self.selectedCrashesIds.length), "red", function(result) {
-                        if (!result) {
-                            return true;
-                        }
-                        countlyCrashes.resolving(self.selectedCrashesIds, function(data) {
-                            if (!data) {
-                                CountlyHelpers.alert(jQuery.i18n.map["crashes.try-later"], "red");
-                            }
-                            else {
-                                self.resetSelection(true);
-                            }
+            },
+            stopEditingComment: function() {
+                this.$data.commentInput = "";
+                this.$data.commentBeingEdited = undefined;
```

Comment:
Why not `this.commentInput` and instead `this.$data.commentInput`?

Reply:
I guess it looked more consistent near `this.$otherStuff`. Is there a semantical difference?